docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  ollama-container:
+    image: ollama/ollama
+    volumes:
+      - ./data/ollama:/root/.ollama
+    ports:
+      - 11434:11434
+
+  langchain-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./app:/app
```

Have you tried [using `depends_on` instead of a `time.sleep` to let docker compose handle the wait time until ollama-container is ready](https://docs.docker.com/compose/compose-file/05-services/#depends_on)?
```suggestion
      - ./app:/app
    depends_on:
      - ollama-container
```